packages/sources/onchain-gas/src/adapter.ts

```diff
@@ -28,9 +28,14 @@ export const makeWSHandler = (config?: Config): MakeWSHandler => {
       connection: {
         url: defaultConfig.api.baseURL,
       },
-      toSaveFromFirstMessage: (message: any) => ({
-        subscriptionId: message.params.subscription,
-      }),
+      toSaveFromFirstMessage: (message: any) => {
```

Comment:
Could we add a comment explaining a little bit about why this is used and when it happens?

Reply:
There's a comment in the `index.d.ts` file.